NuRadioReco/framework/parameters.py

```diff
@@ -36,9 +36,11 @@ class stationParameters(Enum):
     cr_xmax = 28  #: Depth of shower maximum of the air shower
     vertex_2D_fit = 29  #: horizontal distance and z coordinate of the reconstructed vertex of the neutrino
     distance_correlations = 30
-    shower_energy = 31 #: the energy of the shower 
+    shower_energy = 31 #: the energy of the shower
     viewing_angles = 32 #: reconstructed viewing angles. A nested map structure. First key is channel id, second key is ray tracing solution id. Value is a float
-
+    flagged_channels = 60  #: a set of flagged channel ids
+    cr_dominant_polarisation = 61  #: int denoting the group_id of the dominant polarisation in the cosmic ray signal
```

Comment:
We will change the interpretation of `channel_group_id` (as discussed), so this will be just an integer unrelated to that. I added the modules where the parameters are set/calculated.